### DISCUSSION TITLE
Expose loadbalancer IPv4 when Traefik disabled.

### DISCUSSION THREAD

**Discussion by geraldwuhoo** [mysticaltech/terraform-hcloud-kube-hetzner#284] [Ideas]
Currently the code in `output.tf` returns `null` for `load_balancer_public_ipv4` if Traefik is disabled. I can understand the reasoning, because without an ingress controller to provision an IP from the Hetzner LB, there is no LB IP to speak of. However, I'm disabling Traefik to install nginx-ingress instead. I'm sure many others disable Traefik to instead install their own preferred ingress controller.

Since there is no way for the Terraform module to detect whether or not a user will provision an ingress controller *after* Terraform provisioning, I think it makes sense to err on the side of caution, and return proper `load_balancer_public_ipv4` value even when Traefik is disabled.

**Comment by mysticaltech** [comment]
Hello @geraldwuhoo, it makes sense! PRs welcome, that should be a quick fix.

@phaer FYI.

**Comment by mysticaltech** [comment]
Moving this to a discussion in the ideas section, @geraldwuhoo, If you decide to take this on, I will convert it back to an issue to properly close it and give proper attributions.

**Comment by mysticaltech** [comment]
Implemented in PR: https://github.com/mysticaltech/terraform-hcloud-kube-hetzner/pull/2076

Closing as addressed for the v3 workstream.
